### ISSUE TITLE
Is there an error in the discriminator's label?

### ISSUE THREAD

**Issue by wanghy1997** [cubeyoung/TA-VAAL#5]
There is a logical error in 'selection_methods.py', can you explain the purpose of doing this?
"
<img width="675" alt="Image" src="https://github.com/user-attachments/assets/4d9d2a24-8be2-4e30-8377-d21306d2acd0" />
"
lab_real_preds = torch.ones(labeled_imgs.size(0))
unlab_fake_preds = torch.zeros(unlabeled_imgs.size(0))
Why are all the tags 1, rather than one being 1 and the other being 0?
